webdriver/tests/new_window/new_tab.py

```diff
@@ -26,23 +26,32 @@ def test_new_tab_opens_about_blank(session):
     value = assert_success(response)
     assert value["type"] == "tab"
 
-    session.handle = value["handle"]
-    assert session.url == "about:blank"
+    original_handle = session.window_handle
+    session.window_handle = value["handle"]
+    new_window_url = session.url
+    session.window_handle = original_handle
```

Comment:
Why are you switching back to the original handle? I don't see that this is necessary.

Reply:
Does the framework do so automatically? I didn’t pay close enough attention to what browsing contexts got closed at the end of each subtest. If it’s unneeded, I can easily adjust.

Reply:
Yes, we run the `cleanup_session()` method from fixtures.py after each test, which cleans everything up.

Reply:
Alright then, I’ll make the changes to revert back to the earlier assert location and behavior.

Reply:
Fixed now.